docs/usageScenarios/automatic_reconstruction_scenario.rst

```diff
@@ -0,0 +1,10 @@
+For this scenario, as soon as a full set of 3D acquisition is done (when all the rotation angle have been acquired)
+iMars3D should be automatically triggered and reconstruct the data and export them into the IPTS-XXXX/shared/processed_data folder.
+
+This scenario used to work and stopped working 2 years ago. Script used can be find in `/HFIR/CG1D/shared/autoreduce`
+
```

Comment:
Please add a command line example with specific files it could be run against. Also remove the empty lines at the end of the file.

Reply:
Same files as the ones provided in the command line scenario can be used. 

@yxqd , can you help us here providing a command-line example that they can used to test the implementation.